lib/Draggable.es6

```diff
@@ -138,6 +138,13 @@ export default class Draggable extends DraggableCore {
     }
   }
 
+  componentWillReceiveProps(nextProps) {
```

Comment:
This would cause the draggable to jump if you changed any property at all, or even if a render cascaded down to this element.